.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: java
+jdk:
+    - oraclejdk7
```

Comment:
I believe we specified JDK 6 or higher in the non-functional requirements. please change.